webpack.config.js

```diff
@@ -20,30 +18,20 @@ module.exports = {
       {
         test: /\.scss$/i,
         use: [
-          MiniCssExtractPlugin.loader,
+          'style-loader',
           {
             loader: 'css-loader',
             options: {
               esModule: true,
               modules: {
+                auto: true,
```

Comment:
this config value will detect global vs module CSS based on filename: https://github.com/webpack-contrib/css-loader#auto

so we can remove the duplicative global SCSS rule below